plugin/tests/unit/sdk_v2/extension/test_sdk_v2_extension_surface.py

```
@@ -0,0 +1,131 @@
+    assert extension.Result is not None
+    assert extension.ErrorCode is not None
+
+
+def test_extension_decorator_metadata_exports() -> None:
+    assert dec.EXTENSION_ENTRY_META == "__extension_entry_meta__"
+    assert dec.EXTENSION_HOOK_META == "__extension_hook_meta__"
+    assert dec.ExtensionEntryMeta(id="x", name=None, description="", timeout=None).id == "x"
+    assert dec.ExtensionHookMeta(target="*", timing="before", priority=0).timing == "before"
+
+
+def test_extension_proxy_object_forwards(monkeypatch: pytest.MonkeyPatch) -> None:
```

Ruff (0.15.5)

[warning] Unused function argument: `monkeypatch`

(ARG001)